app/views/JitsiMeetView.js

```diff
@@ -30,39 +31,45 @@ class JitsiMeetView extends React.Component {
 	constructor(props) {
 		super(props);
 		this.rid = props.route.params?.rid;
-		this.onConferenceTerminated = this.onConferenceTerminated.bind(this);
-		this.onConferenceJoined = this.onConferenceJoined.bind(this);
+		this.url = props.route.params?.url;
 		this.jitsiTimeout = null;
-	}
 
-	componentDidMount() {
-		const { route, user, baseUrl } = this.props;
+		const { user, baseUrl } = props;
 		const {
 			name: displayName, id: userId, token, username
 		} = user;
-
 		const avatarAuthURLFragment = `&rc_token=${ token }&rc_uid=${ userId }`;
 		const avatar = formatUrl(username, baseUrl, 100, avatarAuthURLFragment);
-
-		setTimeout(() => {
-			const userInfo = {
+		this.state = {
+			userInfo: {
 				displayName,
 				avatar
-			};
-			const url = route.params?.url;
-			const onlyAudio = route.params?.onlyAudio ?? false;
-			if (onlyAudio) {
-				JitsiMeet.audioCall(url, userInfo);
-			} else {
-				JitsiMeet.call(url, userInfo);
 			}
-		}, 1000);
+		};
+	}
+
+	componentDidMount() {
+		const { route } = this.props;
+		const { userInfo } = this.state;
+
+		if (isIOS) {
```

Comment:
I'm calling Jitsi on iOS the old way and Android using props.
For some reason I didn't understand, Android is not working this way anymore.